src/common/telemetry/src/logging.rs

```diff
@@ -82,6 +83,10 @@ pub struct LoggingOptions {
 
     /// The protocol of OTLP export.
     pub otlp_export_protocol: Option<OtlpExportProtocol>,
+
+    /// Additional HTTP headers for OTLP exporter.
+    #[serde(skip_serializing_if = "HashMap::is_empty")]
+    pub otlp_headers: HashMap<String, String>,
```

Comment:
```suggestion
    #[serde(default)]
    #[serde(skip_serializing_if = "HashMap::is_empty")]
    pub otlp_headers: HashMap<String, String>,
```
Let's add `#[serde(default)]` too for default deserialization